src/style.css

```diff
@@ -0,0 +1,30 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
you can add only box-sizing to the `*{  }`, other styles you can add to tag body